cmd/jaeger/internal/integration/trace_writer.go

```diff
@@ -81,7 +81,7 @@ func (w *traceWriter) WriteTraces(ctx context.Context, td ptrace.Traces) error {
 			scope    ptrace.ScopeSpans
 			resource ptrace.ResourceSpans
 		)
-		
+
```

Comment:
I've been running into this too when running `make fmt`. I'm not sure why CI doesn't complain but it gets fixed when running `make fmt` locally.

Reply:
We may have lost this in some refactoring. We do have linters (e.g. `./scripts/lint/updateLicense.py`) that can modify the files and then fail if there were changes. We could have the same with a lint step that runs `make fmt` and fails if there's a diff.